### DISCUSSION TITLE
Why isn't Automa switching tabs

### DISCUSSION THREAD

**Discussion by tab-switcher** [AutomaApp/automa#1314] [Q&A]
My workflow is:
1. Active tab
2. Open new tab (https://translate.google.com/) - **works**
3. Switch tab (match patterns, https://cloud.memsource.com/) - **nothing happens**
4. Get text **(I assume this will copy the text in an element - right?)**
5. Switch tab (match patterns, https://translate.google.com/)
6. Click element
7. Press key (CTRL+V)
8. Get text
9. Switch tab (match patterns, https://cloud.memsource.com/)
10. Click element
11. Press key (CTRL+V

This is supposed to copy text from Phrase, paste into Google Translate, then copy the translation generated in Google Translate, then paste it into the empty translation segment in Phrase. I know it isn't good rn but I would like to know why it isn't switching tabs specifically in step 3, so I can move on to fine tuning the remaining steps.

Screenshots:

![image](https://github.com/AutomaApp/automa/assets/142053539/d4f604d9-0d5f-49f2-afcc-5ba5b7bc12f7)
![image](https://github.com/AutomaApp/automa/assets/142053539/2884de36-0d90-4544-aaf5-204d9d2a88a6)

Many thanks in advance.

**Comment by Kholid060** [comment]
It's not working because you're not specified to match the URL path in your match pattern. To fix this, add `*` at the end of the match pattern like `https://cloud.memsource.com/*`; See [this page](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Match_patterns#examples) to learn more about match pattern
